tests/apps/search/test_query_courses.py

```diff
@@ -0,0 +1,612 @@
+"""
```

Comment:
This is the test suite that cannot be ran on its own `bin/pytest tests/apps/search/test_query_courses.py`

Reply:
Ok there was some incoherence in imports. I refactored a bit more :recycle: